config/routes.rb

```diff
@@ -409,7 +409,9 @@
     jsonapi_resources :anime
     jsonapi_resources :manga
     jsonapi_resources :drama
-    jsonapi_resources :users
+    jsonapi_resources :users do
+      jsonapi_resources :bestowments, only: [:index]
```

Comment:
It's preferable to keep everything at the top level I think, JR will automatically add routes for the association